content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -292,9 +295,11 @@ be co-located in the same defined topology, eg., the same node.
 
 In a three node cluster, a web application has in-memory cache such as redis. We want the web-servers to be co-located with the cache as much as possible.
 
-Here is the yaml snippet of a simple redis deployment with three replicas and selector label `app=store`. The deployment has `PodAntiAffinity` configured to ensure the scheduler does not co-locate replicas on a single node.
+Here, below there is the yaml snippet of a simple redis deployment with three replicas and selector label `app=store`. The deployment has `PodAntiAffinity` configured to ensure the scheduler does not co-locate replicas on a single node.
```

Comment:
👋 @r0ushann . 
I suggest to remove the phrase ", below there" from the sentence.